ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -19,6 +19,8 @@ let prototyped_of_field = function
       Some "22.27.0"
   | "host", "last_software_update" ->
       Some "22.20.0"
+  | "pool", "bias_enabled" ->
```

Comment:
This change is out of date, 22.34.0 has been released. Please change it by removing changes in this file and rebuilding xapi